yt/yt/tests/integration/master/test_node_tracker.py

```diff
@@ -449,3 +449,51 @@ def get_online_node_count():
         wait(lambda: get_online_node_count() == 1)
         sleep(5)
         assert get_online_node_count() == 1
+
+
+################################################################################
+
+
+class TestRackDataCenter(YTEnvSetup):
+    NUM_MASTERS = 1
+    NUM_NODES = 1
+    NUM_SCHEDULERS = 1
+    rack_set = "rack"
+    data_center_set = "data_center"
```

Comment:
I'm not sure about this and it's better to look around but I believe that class-level variables are always capitalized, even when don't have a special meaning to `YTEnvSetup`.